02-Reproject-Raster-In-R.Rmd

```diff
@@ -246,15 +258,16 @@ plot(DTM_HARV,
      legend=F)
 ```
 
-We have now successfully layered the Digital Terrain Model on top of our hillshade
-to produce a nice looking, textured basemap! 
+We have now successfully layered the Digital Terrain Model on top of our
+hillshade to produce a nice looking, textured base map! 
+
+##Challenge
```

Comment:
See shared Evernote NEON Data Skills Format Questions for response on formatting.